### PR TITLE
Convert noteblocks for glossary folder

### PR DESCRIPTION
**Pull request by queengooborg**
This PR converts the noteblocks for the 'glossary' folder to GFM syntax, using a [conversion script](https://github.com/queengooborg/mdn-toolkit/blob/main/upgrade-noteblock.js).

### CODE REVIEW

**Review by Josh-Cena — APPROVED** [review]

File: files/en-us/glossary/global_object/index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ page-type: glossary-definition
 
 The **global object** in JavaScript is an {{glossary("object")}} which represents the {{glossary("global scope")}}.
 
-> **Note:** _Globally available objects_, which are objects in the {{glossary("global scope")}}, are sometimes also referred to as global objects, but strictly speaking, there is only one global object per environment.
+> [!NOTE] > _Globally available objects_, which are objects in the {{glossary("global scope")}}, are sometimes also referred to as global objects, but strictly speaking, there is only one global object per environment.
```
Comment (Josh-Cena) [inline files/en-us/glossary/global_object/index.md:11]:
```suggestion
> [!NOTE]
>
> _Globally available objects_, which are objects in the {{glossary("global scope")}}, are sometimes also referred to as global objects, but strictly speaking, there is only one global object per environment.
```

File: files/en-us/glossary/cors-safelisted_response_header/index.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ By default, the safelist includes the following response headers:
 
 Additional headers can be added to the safelist using {{HTTPHeader("Access-Control-Expose-Headers")}}.
 
-> **Note:** {{HTTPHeader("Content-Length")}} was not part of the original set of safelisted response headers \[[ref](https://github.com/whatwg/fetch/pull/626)]
+> [!NOTE] > {{HTTPHeader("Content-Length")}} was not part of the original set of safelisted response headers \[[ref](https://github.com/whatwg/fetch/pull/626)]
```
Comment (Josh-Cena) [inline files/en-us/glossary/cors-safelisted_response_header/index.md:23]:
```suggestion
> [!NOTE]
>
> {{HTTPHeader("Content-Length")}} was not part of the original set of safelisted response headers \[[ref](https://github.com/whatwg/fetch/pull/626)]
```

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
@queengooborg There is a rendering difference with an extra blank line:

image

And I don't find it acceptable to make this change, because it doesn't appear revertible to me (once we make this workaround it would be hard to discover all places that are intended as workaround, vs. those intended to start a new paragraph). We may have to wait on the Prettier bug.

In your conversion script, could you not do conversion for any note that starts with non-letters?

In the meantime, I think we should revert all notes with the `> \[![A-Z]+\]\n>\n` form to the previous state (except callouts, which were supposed to look like that). There aren't many on main.

File: files/en-us/glossary/global_object/index.md (hunk 1 above)
Comment (queengooborg) [reply files/en-us/glossary/global_object/index.md]: Thanks for catching that!  Looks like there's more special characters than I thought, haha!

**Comment by queengooborg** [comment]
Agh, okay, there _is_ a difference in rendering, whoops -- I genuinely thought that I had added logic to make the rendering identical with a newline in between...  🤦‍♀️ 

> In your conversion script, could you not do conversion for any note that starts with non-letters?

I could make that change, but, uh...I already ran the script on all the files locally, fixed issues related to Prettier/Markdownlint malformatting, and submitted PRs for the vast majority of the docs...  😅 

---

Truth be told, I don't think that this rendering difference is too much of an issue if it was styled better (AKA, more like how GitHub renders the noteblocks).  But that's just my opinion and would be something that should be discussed before proceeding with!

**Comment by Josh-Cena** [comment]
@queengooborg Fear not, I'm checking out each of your branches and running a regex search on each one, so you can just look at the cases I pointed out.

> Truth be told, I don't think that this rendering difference is too much of an issue if it was styled better (AKA, more like how GitHub renders the noteblocks). But that's just my opinion and would be something that should be discussed before proceeding with!

Agreed; I would be equally happy if we render like GitHub (put "Note" on a separate line).

File: files/en-us/glossary/global_object/index.md [hunk 3]
```diff
@@ -8,7 +8,9 @@ page-type: glossary-definition
 
 The **global object** in JavaScript is an {{glossary("object")}} which represents the {{glossary("global scope")}}.
 
-> **Note:** _Globally available objects_, which are objects in the {{glossary("global scope")}}, are sometimes also referred to as global objects, but strictly speaking, there is only one global object per environment.
+> [!NOTE]
+>
+> _Globally available objects_, which are objects in the {{glossary("global scope")}}, are sometimes also referred to as global objects, but strictly speaking, there is only one global object per environment.
```
Comment (Josh-Cena) [inline files/en-us/glossary/global_object/index.md:13]:
```suggestion
> **Note:** _Globally available objects_, which are objects in the {{glossary("global scope")}}, are sometimes also referred to as global objects, but strictly speaking, there is only one global object per environment.
```

**Review by Josh-Cena — APPROVED** [review]

**Comment by hochan222** [comment]
@queengooborg Hello. 

I recently confirmed this change in the `locale-ko`. Is there a locale guide?

**Comment by queengooborg** [comment]
There isn't really a guide for performing the conversion, even for English content -- but there is a script I wrote, which can be found in the description of this PR!

**Comment by hochan222** [comment]
I asked because I wanted to change NOTE to Korean, but it looks like I should use NOTE as is!

I understand. I am leaving some of the contents of the MDN Markdown guide for other people. Thank you :)

image

- https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN

```
> [!NOTE]  
> Highlights information that users should take into account, even when skimming.

> [!TIP]
> Optional information to help a user be more successful.

> [!IMPORTANT]  
> Crucial information necessary for users to succeed.

> [!WARNING]  
> Critical content demanding immediate user attention due to potential risks.

> [!CAUTION]
> Negative potential consequences of an action.
```

**Comment by queengooborg** [comment]
That's correct -- the magic keywords should stay in English now, rather than being localized.  This ensures consistency across locales and better compatibility with Markdown renderers besides Yari (like GitHub's web editor)!

**Comment by hochan222** [comment]
> Agreed; I would be equally happy if we render like GitHub (put "Note" on a separate line).

@queengooborg, @Josh-Cena How about adding prettier ignore?

```
<!-- prettier-ignore-start -->
> [!NOTE]
> [Here is another example without Canvas or external libraries.](https://jsfiddle.net/jlr7245/teb4znk0/20/)
<!-- prettier-ignore-end -->
```

**Comment by Josh-Cena** [comment]
I don't find it necessary and it adds more noise than just keeping it unmigrated.

**Pull request merged**